Common/FeatureFlags.swift

```diff
@@ -193,6 +195,8 @@ struct FeatureFlagConfiguration: Decodable {
         #else
         self.usePositiveMomentumAndRCForManualBoluses = true
         #endif
+
+        self.dynamicCarbAbsorptionEnabled = true
```

Comment:
Yeah, this could probably be cleaned up; it's a longstanding code based config change (which I generally don't want to keep), but trying to minimize unrelated changes, as this is a big changeset already.